src/app/zones/data/index.ts

```diff
@@ -1,6 +1,16 @@
-import type { ZoneOverview, StatusKeyword } from '@/types/index.d'
+import type { ZoneOverview } from '@/types/index.d'
 import { get } from '@/utilities/get'
 
+// TODO(jc): These two are extremely similar, see if we can merge
+export function getZoneControlPlaneEnvironment(zoneOverview: ZoneOverview): string {
+  // TODO(jc): is it ok to use the config from the first subscription we find here?
```

Comment:
**Comment**: I think we should generally only look at the last subscription, period. That reduces the risk of making false assumptions. The last one is the most recent and should be the only thing we care about.